docs/index.md

```diff
@@ -0,0 +1,15 @@
+# http4s-session 
+![Continuous Integration](https://github.com/http4s/http4s-session/workflows/Continuous%20Integration/badge.svg) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.http4s/http4s-session_2.13/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.http4s/http4s-session_2.13) ![Code of Consuct](https://img.shields.io/badge/Code%20of%20Conduct-Scala-blue.svg)
+
+Session Management for http4s.
+
+## Quick Start
+
+To use http4s-session in an existing SBT project with Scala 2.13 or a later version, add the following dependencies to your
+`build.sbt` depending on your needs:
+
+```scala
+libraryDependencies ++= Seq(
+  "org.http4s" %% "http4s-session" % "<version>"
```

Comment:
```suggestion
  "org.http4s" %% "http4s-session" % "@VERSION@"
```

Reply:
That's interesting but `@VERSION@` uses a snapshot version. Do you know if we can bypass that behaviour?

Reply:
Anyway, I'm not sure that this microsite will be published, so whatever, let's move on

Reply:
Hmm, that's bizarre that I saw the snapshot version locally, but at https://http4s.github.io/http4s-session/ we are good!